src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -3596,6 +3597,13 @@ emit_arm64_intrinsics (
 				(is_64bit ? OP_XOP_I8_I8 : OP_XOP_I4_I4),
 				(is_64bit ? INTRINS_BITREVERSE_I64 : INTRINS_BITREVERSE_I32),
 				arg0_type, fsig, args);
+		case SN_Yield: {
+			MonoInst* ins;
+			MONO_INST_NEW (cfg, ins, OP_NOP);
```

Comment:
arm64 has a yield instruction.

Reply:
Added `OP_ARM64_HINT` to account for this and other hint codes. This is also reflected on LLVM side.